packages/iris-grid/src/IrisGridTreeTableModel.ts

```diff
@@ -230,6 +230,12 @@ class IrisGridTreeTableModel extends IrisGridTableModelTemplate<
     return [this.virtualColumns.length, this.groupedColumns.length];
   }
 
+  get layoutHints(): DhType.LayoutHints | null | undefined {
+    // TODO: Update this when JS API types are updated with this.
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    return (this.table as any).layoutHints;
```

Comment:
Should we feature detect here? If this had to be added to the DHC JSAPI then it's probably not part of the DHE JSAPI

Although since it's just 1 level deep it shouldn't throw and should just return undefined like it would have before

Reply:
Yea you're right, I should be checking here.